class-01-jquery-and-the-dom/pair-assignment/ron-yunjoo/scripts/article.js

```diff
@@ -1,27 +1,30 @@
 var articles = [];
 
 function Article (opts) {
-  // TODO: Use the js object passed in to complete this contructor function:
+  // todo: Use the js object passed in to complete this contructor function:
   // Save ALL the properties of `opts` into `this`.
+  this.title =opts.title;
   this.author = opts.author;
   this.category = opts.category;
-  this.authorUrl = opts.uthorUrl;
-  this.title = opts.title;
-  this.body = opts.body;
   this.publishedOn = opts.publishedOn;
+  this.authorUrl = opts.publishedOn;
+  this.body = opts.body;
 }
 
 Article.prototype.toHtml = function() {
   var $newArticle = $('article.template').clone();
 
   $newArticle.data('category', this.category);
-  
-  // TODO: Use jQuery to fill in the template with properties
+  $newArticle.find('h1').text(this.title);
+  $newArticle.find('.article-body').html(this.body);
+  $newArticle.find('a[href]').attr('href',this.authorUrl);
```

Comment:
this part isn't working for me, is it working for you? try:
`$newArticle.find('a').attr('href', this.authorUrl);`